app/src/main/java/org/wikipedia/dataclient/Service.java

```diff
@@ -339,5 +339,21 @@
     Observable<MwPostResponse> postReviewImageLabels(@NonNull @Field("filename") String fileName,
                                                      @NonNull @Field("token") String token,
                                                      @NonNull @Field("batch") String batchLabels);
+    // ------- Watchlist -------
 
+    @GET(MW_API_PREFIX + "action=query&format=json&list=watchlist&continue=")
+    @NonNull Observable<MwQueryResponse> getWatchlist(@Nullable @Query("continue") String cont);
+
+    @Headers("Cache-Control: no-cache")
+    @POST(MW_API_PREFIX + "action=watch&format=json")
+    @FormUrlEncoded
+    Observable<EntityPostResponse> unWatch(@Field("unwatch") int unwatch,
```

Comment:
In this case you can just put `&unwatch=1` in the annotation, instead of as a parameter.